### ISSUE TITLE
A WEP downgrade attack with poc !

### ISSUE THREAD

**Issue by minanagehsalalma** [wifiphisher/wifiphisher#1116]
https://www.wifi-libre.com/topic-756-una-historia-de-rogue-ap-el-pdf-de-koala-traducido-al-espanol.html

![56949205_396899957568775_6957832214767730688_n](https://user-images.githubusercontent.com/20546638/56146299-46e2f680-5fa6-11e9-9695-59cd47f08e28.jpg)

**Comment by sophron** [comment]
Yes, we can potentially add support for downgrade attacks to `roguehostapd`.

**Comment by minanagehsalalma** [comment]
@sophron thanks that would be great

**Comment by djdisikd** [comment]
Force the client to change to WEP encryption?

**Comment by minanagehsalalma** [comment]
@djdisikd  not really. . ... we create a rouge WEP instead  with opn encryption  ....
Read the article mate

**Comment by djdisikd** [comment]
But what if the victim uses the password length of 9 ?

How is it possible to connect to the rogue WEP network?

Note: You can't connect to a WEP network with password length of 9

**Comment by minanagehsalalma** [comment]
@djdisikd "You can't connect to a WEP network with password length of 9" 
So said that ? 
You can ....

You should test it yourself before saying such things !!

**Comment by djdisikd** [comment]
Are you kidding me little kid, go and look at the internet.

You said can right? Where is the evidence? Huh?

Can you use your brain before saying it out.

And you are the one who should test it yourself.

Please control your attitude, you are very rude

**Comment by minanagehsalalma** [comment]
@djdisikd 
I am very rude ? What did I say that is rude ?
Check the link mate 
https://askubuntu.com/questions/304460/wifi-only-accepts-passwords-of-5-or-13-characters
'Where is the evidence? "
Where is your evidence ?!! At least I provided a link that explains what I am aiming to !
"And I test it before, do you want me to show you?" 
Yep show me but  against windows or android .
If you have checked the link in the first place you do find that it says to break a 26 hexidecimals  that means 13 ASCII

**Comment by djdisikd** [comment]
If you read the link, why did you say WEP supports 9 characters?

**Comment by minanagehsalalma** [comment]
Who said that if the network supports a maximum length of 13 ASCII .   can't support  9 ASCII !?

And it very Depends on the number of bits you use 
![IMG_20190421_013558](https://user-images.githubusercontent.com/20546638/56463524-75ae0200-63d6-11e9-8aad-956e013a3953.png)
@djdisikd

**Comment by djdisikd** [comment]
Nope, it doesn't support 9 ASCII

All the number of bits doesn't support 9 ASCII

**Comment by minanagehsalalma** [comment]
@djdisikd  that's weird why higher number of bits doesn't support the length of lower bits !
Is the maximum length is the minimum in wep ?
Let's  change the ssid name to "add one number to your 9 characters password please *
And hope that he follows the ssid   advise 😅

**Comment by djdisikd** [comment]
Wi-Fi alliance who created WEP, you should ask them

Maximum length is the minimum of wep? ( don't understand)

But if you change the ssid name the owner will think that the Wi-Fi is others not hers

**Comment by minanagehsalalma** [comment]
"(don't understand)"
Who don't understand  ?
Me . Maybe
"But if you change the ssid name the owner will think that the Wi-Fi is
others not hers"

I was just joking   ..  btw signal strength  plays a big role in this
ether  do the mkd3

**Comment by minanagehsalalma** [comment]
@djdisikd  I think there is still a way around this but it requires higher skills
https://www.quora.com/Is-it-possible-to-create-a-WEP-wifi-that-accepts-any-password

**Comment by djdisikd** [comment]
Maybe, I am not sure about that

**Comment by minanagehsalalma** [comment]
neither do I

**Comment by minanagehsalalma** [comment]
BTW you are not just not allowed to connect with these lengths like 9 ..
You are not allowed to click connect  till you adjust the key length so I
think he mate do it himself

**Comment by djdisikd** [comment]
ok

**Comment by abdlhay** [comment]
The provided link no longer works couldn't find POC anywhere around the internet, can anyone please tell me where I can start and how exactly this method works in details
